chesster.js

```diff
@@ -161,6 +152,17 @@ function update_channels(bot){
     });
 
 }
+function refreshCurrentRoundSchedules() {
+    var _45_45 = league.getLeague("45+45", config);
+    _45_45.getCurrentRoundSchedule(function(err, pairings) {
```

Comment:
Actually, I have changed my mind. I like your version.

While getRosters doesn't _need_ to run every 30s, we don't know when the mods will actually update the rosters, so we can't time the refresh to happen immediately after that. So it's probably fine to just update all three at the same interval